src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.ServiceClientAdapter/Commands.RecoveryServices.Backup.ServiceClientAdapter.csproj

```diff
@@ -97,6 +97,9 @@
       <SubType>Designer</SubType>
     </None>
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="Microsoft.Azure.Management.RecoveryServices.Backup.dll" />
```

Comment:
Looks like the DLL was introduced in a previous PR and so this is not reflected in this PR.